pyproject.toml

```diff
@@ -42,6 +42,7 @@ Mako = "*"
 Markdown = "*"
 MarkupSafe = "*"
 Mercurial = "*"
+mrcfile = "==1.2.0"
```

Comment:
Is there a reason for this pin ?

Reply:
It's just the current version, and I noticed that other requirements were pinned.  I will remove it.

Reply:
These are only pinned if we know that's necessary. The toml file is used to update and generate the requirements.txt files that are always fully pinned.